v1.0/timestamp.md

```diff
@@ -4,13 +4,30 @@ summary: The TIMESTAMP data type stores a date and time pair in UTC, whereas TIM
 toc: false
 ---
 
-The `TIMESTAMP` [data type](data-types.html) stores a date and time pair in UTC, whereas `TIMESTAMPTZ` stores a date and time pair with a time zone offset from UTC. 
+The `TIMESTAMP` [data type](data-types.html) stores a date and time pair in UTC.
 
 <div id="toc"></div>
 
+## Time Zone Details
+
+`TIMESTAMP` has two variants:
+
+- `TIMESTAMP WITH TIME ZONE` converts `TIMESTAMP` values from UTC to the client's session time zone (unless another time zone is specified for the value). However, it is conceptually important to note that `TIMESTAMP WITH TIME ZONE` *does not* store any time zone data.
+
+- `TIMESTAMP WITHOUT TIME ZONE` presents all `TIMESTAMP` values in UTC.
+
+The difference between these two types is that `TIMESTAMP WITH TIME ZONE` uses the client's session time zone, while the other simply does not. This behavior extends to functions like `now()` and `extract()` on `TIMESTAMP WITH TIME ZONE` values.
```

Comment:
If we aren't going to note here that the session time zone defaults to UTC, can we at least provide a link to the SET TIME ZONE section? Users need a fast way to discover the default.